bootstrap/templates/new/mix.exs

```diff
@@ -59,7 +59,8 @@ defmodule <%= app_module %>.Mixfile do
   # Specify target specific dependencies
   def deps("host"), do: []
   def deps(target) do
-    [{:"nerves_system_#{target}", ">= 0.0.0"}]
+    [{:nerves_runtime, "~> <%= runtime_vsn %>"},
+     {:"nerves_system_#{target}", ">= 0.0.0", runtime: false}]
```

Comment:
Good point, but we still do release them in lock-step, with only the patch-level being potentially different, right? For example, we'd set the constraint to `"~> 0.10.0"` today.